pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -179,6 +182,17 @@ type SeedControllerConfiguration struct {
 	SyncPeriod *metav1.Duration `json:"syncPeriod,omitempty"`
 }
 
+// ExtensionsStateSynchronizationControllerConfiguration dsa
+type ExtensionsStateSynchronizationControllerConfiguration struct {
```

Comment:
Proper defaulting is missing in `defaults.go`. Also, the doc string seems broken.

Reply:
Additionally the comment needs to be fixed: "dsa" seems out of place:)